### DISCUSSION TITLE
Show hidden/dot files

### DISCUSSION THREAD

**Discussion by omnigenous** [gokcehan/lf#1735] [Q&A] [answered]
I tried adding `set hidden false` but it doesn't seem to change anything

**Comment by omnigenous** [comment]
Also is it possible to set key bind to toggle on/off showing hidden files?

**Comment by DusanLesan** [comment, marked as answer]
```
set hidden true
```
Is used to show the hidden files. False is by [default](https://github.com/gokcehan/lf/blob/master/doc.md#quick-reference). Keys `zh` are default key-binding but you can set it like `map . set hidden!`
